fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -262,6 +262,9 @@
      */
     public static final String NT_VERSION_FILE = "nt:versionFile";
 
+    // VERSIONING
+    public static final String VERSIONING_TIMEMAP_TYPE = REPOSITORY_NAMESPACE + "TimeMap";
```

Comment:
This looks good.
Could you also take this opportunity to move https://github.com/fcrepo4/fcrepo4/blob/memento-versioning/fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java#L174 down to this `// VERSIONING` section... and use a constant for the repository namespace?